.github/ISSUE_TEMPLATE/membership.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Organization Membership Request
+about: Request membership in a Argoproj Org
+title: 'REQUEST: New membership for <your-GH-handle>'
+labels: area/github-membership
+assignees: ''
+
+---
+
+### GitHub Username
+e.g. (at)example_user
+
+### Organization you are requesting membership in
+e.g. (at)argoproj
+
+### Requirements
+- [ ] I have reviewed the community membership guidelines (https://github.com/argoproj/argoproj/blob/master/community/membership.md)
+- [ ] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
+- [ ] I am actively contributing to 1 or more Argoproj subprojects
+- [ ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
+- [ ] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application
+
+### Sponsors
+- (at)sponsor-1
+- (at)sponsor-2
+
+### List of contributions to the Argoproj project
+- PRs reviewed / authored
+- Issues responded to
```

Comment:
This can also be feature/enhancement proposals?

Reply:
I left it as is since I'm not sure that simply filing issues is a path toward membership, but actually resolving issues via PRs or giving a solution.